modules/olm-operator-framework.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * operators/what-are-operators.adoc
+
+[id='olm-operator-framework-{context}']
+= Operator Framework
+
+The Operator Framework is a family of tools and capabilities to deliver on the
+customer experience describe above. It is not just about writing code. Testing,
+delivery, and updating Operators are just as important. The Operator Framework
```

Comment:
Should "delivery" be "delivering"? Or "delivery of"?

Reply:
Good question. I think "are" is proper, but I agree that "is" sounds better. Maybe since we're comparing this grouping of 3 things to the single thing about writing code. I couldn't tell you which is truly correct though.